src/osd/ReplicatedBackend.cc

```diff
@@ -1851,7 +1852,9 @@ bool ReplicatedBackend::handle_pull_response(
       pull_info.lock_manager);
   }
 
-
+  // if `first` is true, obc was just set above. Otherwise, we should be
+  // able to reuse it.
```

Comment:
@athanatos, couldn't find any comments re `first`. Can you verify I got this one right please? Thanks!

Reply:
I agree that at this point ```pull_info.obc``` *must* be non-null.  If ```first``` is set, then we just populated it.  Otherwise, it's still in ```pull_info``` from when the first chunk was received.  The invariant definitely appears to be that by the time we get here ```pull_info.obc``` is populated.